integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -45,12 +46,88 @@ public void testAddDate() throws IOException {
 
     result =
         executeQuery(String.format(
-            "source=%s | eval f = adddate('2020-09-16 17:30:00', interval 1 day) | fields f", TEST_INDEX_DATE));
+            "source=%s | eval f = adddate('2020-09-16 17:30:00', interval 1 day) | fields f",
+            TEST_INDEX_DATE));
     verifySchema(result,
         schema("f", null, "datetime"));
     verifySome(result.getJSONArray("datarows"), rows("2020-09-17 17:30:00"));
   }
 
+  @Test
+  public void testConvertTZ() throws IOException {
```

Comment:
What is the importance of each test case? You could break them out into separate tests (🔥) or at least document each one.